src/context/auth/authContextController/AuthContextController.tsx

```diff
@@ -37,19 +40,26 @@ export const AuthContextController = ({ children }: AuthContextControllerProps)
     data: user,
     isLoadingAndEnabled,
     isSuccess: isUserSuccess,
-    remove: resetUser,
+    isError,
   } = useUser({
     enabled: !!state.accessToken,
-    onError: () => {
-      dispatch(resetTokens());
-    },
   });
 
+  const resetUser = useCallback(() => {
+    queryClient.removeQueries({ queryKey: authQueries.me().queryKey });
+  }, [queryClient]);
```

Comment:
what do you think about moving this code back to the `useUser` hook? Thanks to that we will keep the previous contract of it and all the code regarding this query will be encapsulated there if needed to be used somewhere else